test/filter.js

```diff
@@ -85,18 +85,18 @@ describe('Bigtable/Filter', function() {
       assert.strictEqual(str, '1');
     });
 
-    it('should convert a buffer to a string', function() {
+    it('should not do anything to a buffer', function() {
       var str1 = 'hello';
       var buffer = Buffer.from(str1);
       var str2 = Filter.convertToRegExpString(buffer);
 
-      assert.strictEqual(str1, str2);
+      assert.deepStrictEqual(buffer, str2);
     });
 
-    it('should convert a non-utf8 buffer to a string', function() {
+    it('should not incorrectly convert a non-utf8 buffer to a utf8 buffer', function() {
```

Comment:
This test name is confusing and maybe passive-aggressive? :)

Reply:
Changed to be more 😄